### DISCUSSION TITLE
Marketing odo: Release announcements, Articles, Blogs

### DISCUSSION THREAD

**Discussion by valaparthvi** [redhat-developer/odo#5818] [Ideas]
As discussed in the retro call, we want to start announcing new releases on Social Media Platforms, which also involves writing blogs, and more solution oriented articles.

@dharmit I am still unclear on some of the ideas that you had, can you talk more about them here? You mentioned something about creating issues and assigning them during the sprint? What does this include?

**Comment by dharmit** [comment]
I meant to put this up in the Sprint instead of doing it as an ad-hoc task. We could make it a rotating task such that everyone works on writing a release announcement. Meaning, it could be done by you in this Sprint, me in the next, Philippe after that, Armel after Philippe, so on and so forth. Unless, of course, someone's not inclined toward writing (which is absolutely OK!)

As for articles and blogs, I was wondering after the retro about what possible problem statements are we considering writing up for? In the sense, odo is a tool that solves some problems for its users. But what are these problems exactly? We could get some help on this from @serenamarie125, @deboer-tim & @slemeur to come up with user scenarios and then someone can write how odo helps in such a scenario or solves a certain problem.
